### PR TITLE
[changes-to-pix4d-dependabot] Merge the upstream changes.

### PR DESCRIPTION
**Pull request by Pix4D-Janus**
Merge the upstream changes. Check also if Pix4D Dockerfile needs to be updated.

### CODE REVIEW

**Comment by Pix4D-Janus** [comment]
pipeline_setter_bot: your pipeline is set: https://builder.ci.pix4d.com/teams/developers/pipelines/dependabot-upstream_v0.201.0_20220722142348

**Review by aliculPix4D — APPROVED** [review]
I update the Pix4D Dockerfile. It should be fine now.

**Comment by aliculPix4D** [comment]
@Pix4D/integration ready for review

**Review by odormond — APPROVED** [review]

File: Dockerfile.pix4d [hunk 1]
```diff
@@ -33,23 +33,38 @@ RUN apt-get update \
...
+ARG BUNDLER_V2_VERSION=2.3.18
+ENV BUNDLE_SILENCE_ROOT_WARNING=1
+
+# Install Ruby, update RubyGems, and install Bundler
+RUN mkdir -p /tmp/ruby-install \
+ && cd /tmp/ruby-install \
+ && curl -fsSL "https://github.com/postmodern/ruby-install/archive/v$RUBY_INSTALL_VERSION.tar.gz" -o ruby-install-$RUBY_INSTALL_VERSION.tar.gz  \
+ && tar -xzvf ruby-install-$RUBY_INSTALL_VERSION.tar.gz \
+ && cd ruby-install-$RUBY_INSTALL_VERSION/ \
+ && make \
```
Comment (Esysc) [inline Dockerfile.pix4d:61]: For my understanding, why you compile instead of install the compiled package?

**Review by Esysc — APPROVED** [review]

File: Dockerfile.pix4d (hunk 1 above)
Comment (aliculPix4D) [reply Dockerfile.pix4d:61]: These are upstream changes, so I just copy pasted their solution as we had so far.
They first attempt was using rbenv which is simpler in my opinion, I don't know why they choose this way.
But to answer your question, they are doing so so they can install which ever ruby version they want. Also not that they don't install compiled ruby package, just download the instal helper called `ruby-install`

**Pull request merged**